gha/bin/comment-error-to-pr.py

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env python3.10
+import io
+import re
+import mmap
+import glob
+import zipfile
+from argparse import ArgumentParser
+from os import environ
+from typing import NoReturn
+import fileinput
+
+import requests
+from github.WorkflowRun import WorkflowRun
+from github.Workflow import Workflow
+from requests import Response
+
+from gha.github.baseprcommand import BasePrCommand
+from gha.github.basegithubcommand import DEFAULT_GITHUB_TOKEN
+
+DEFAULT_GITHUB_WORKFLOW_RUN = environ.get('GITHUB_RUN_NUMBER') or None
+ERROR_COMMENT_KEY = '## ERROR'
+ERROR_LOGS_KEY = '##[error]'
+LOGS_TIMESTAMP = re.compile(r'(^.*?\s)', re.MULTILINE)
+LOGS_DIR = 'logs'
+
+
+def find_n_replace_in_file(filepath, pattern: re.Pattern, replacer=None) -> str:
+    write = False
+    with fileinput.FileInput(filepath, inplace=True) as file:
+        for line in file:
+            # Strip source code duplication in logs
+            if line.__contains__('shell: /usr/bin/bash') or write is True:
```

Comment:
Сначала простые проверки, потом с вычислениями.

```suggestion
            if write is True or 'shell: /usr/bin/bash' in line:
```